src/config/Blocks/blocks.js

```diff
@@ -363,11 +364,11 @@ const italiaBlocks = {
 };
 
 const getItaliaBlocks = (config) => {
-  delete config.blocks.blocksConfig.teaser;
   config.blocks.blocksConfig.gridBlock.allowedBlocks =
     config.blocks.blocksConfig.gridBlock.allowedBlocks
       .filter((item) => !['slate', 'teaser'].includes(item))
       .concat(['text']);
+  config.blocks.blocksConfig.teaser.schemaEnhancer = italiaTeaserSchemaEnhancer;
```

Comment:
Per non far confusione ai redattori dei vari comuni, e per non dover cambiare tutto il manuale di io-comune, suggerisco di cambiare il title del blocco teaser da 'Teaser' a 'Contenuto in evidenza'